zebra-state/src/lib.rs

```diff
@@ -31,6 +31,19 @@ use zebra_chain::{
 pub mod in_memory;
 pub mod on_disk;
 
+/// The maturity threshold for transparent coinbase outputs.
+///
+/// A transaction MUST NOT spend a transparent output of a coinbase transaction
```

Comment:
Is this part of the protocol?

Reply:
Yes.

Reply:
See page 83 in https://zips.z.cash/protocol/protocol.pdf#consensusfrombitcoin